bigcache_test.go

```diff
@@ -441,6 +441,11 @@ func TestCacheDel(t *testing.T) {
 // TestCacheDelRandomly does simultaneous deletes, puts and gets, to check for corruption errors.
 func TestCacheDelRandomly(t *testing.T) {
 	t.Parallel()
+
+	seed := time.Now().UTC().UnixNano()
+	rand.Seed(seed)
+	print("Seed: ", seed, "\n")
```

Comment:
```suggestion
	t.Printf("Seed: %v", seed)
```

Reply:
There is no such function as `t.Printf` we could use `t.Log` but it's not working where there is a panic

Reply:
Sure, it's `Logf` https://godoc.org/testing#T.Logf

Reply:
Well, but how `print` will in a case of a panic? maybe it's better to place print/log/whatever before `rand.Seed` ?